protocol/bc/txinput.go

```diff
@@ -53,22 +53,21 @@ type (
 var errBadAssetID = errors.New("asset ID does not match other issuance parameters")
 
 func NewSpendInput(txhash Hash, index uint32, arguments [][]byte, assetID AssetID, amount uint64, controlProgram, referenceData []byte) *TxInput {
+	av := uint64(1)
```

Comment:
Why put this in a variable? If anything it should be a `const`, but a literal is best IMO for clarity.